security/middleware.py

```diff
@@ -137,30 +168,36 @@ def load_setting(self, setting, value):
             return
         value = value.lower()
         if value not in XssProtectMiddleware.OPTIONS.keys():
-            raise ImproperlyConfigured(XssProtectMiddleware.__name__ + " invalid option for XSS_PROTECT.")
+            raise ImproperlyConfigured(
+                XssProtectMiddleware.__name__ +
+                " invalid option for XSS_PROTECT.",
+            )
+
         self.option = value
 
     def process_response(self, request, response):
         """
         Add X-XSS-Protection to the reponse header.
         """
-        response['X-XSS-Protection'] = XssProtectMiddleware.OPTIONS[self.option]
+        response['X-XSS-Protection'] = \
```

Comment:
I know this came up in the other PR. I'm at PyCon now and have been thinking about this more. I still don't think this is a good way to format things. The old line was perfectly fine, and I think easier to read. I don't think we should follow the pep blindly. I'd prefer we tell pep8 to ignore long line warnings, and rely on us catching them in code review when they are "too" long.